data/json/recipes/weapon/mods.json

```diff
@@ -722,6 +722,25 @@
     "using": [ [ "cordage_short", 2 ] ],
     "components": [ [ [ "spike", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "axe_mod",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "fabrication",
+    "difficulty": 3,
+    "skills_required": [ [ "cutting", 2 ] ],
+    "time": "10 m",
+    "qualities": [ { "id": "WRENCH", "level": 1 }, { "id": "SAW_M_FINE", "level": 1 }, { "id": "DRILL", "level": 1 } ],
+    "using": [ [ "welding_standard", 1 ] ],
+    "autolearn": true,
+    "components": [
+      [ [ "makeshift_axe", 1 ], [ "ax", 1 ], [ "copper_ax", 1 ], [ "bronze_axe", 1 ], [ "fire_ax", 1 ], [ "crash_axe", 1 ] ],
```

Comment:
By "not steel," do you mean bronze and copper axes? I think it's impossible at file level to achieve a result where the material of the modification will be displayed depending on the material of which the component is made. I see three possible solutions to this problem:

1) Exclude bronze and copper axes from the list of components.
2) Create a separate recipe, for copper and bronze versions.
3) Ignore this nuance, because the modification is non-disassembled and in this case, it is impossible to get "steel" when disassembling. Steel in this case is only a cosmetic description, partially justified by the use of scrap metal in the recipe.